captum/attr/_core/noise_tunnel.py

```diff
@@ -326,7 +331,11 @@ def compute_expected_attribution_and_sq(attribution):
         )
 
     def _apply_checks_and_return_attributions(
-        self, attributions, is_attrib_tuple, return_convergence_delta, delta
+        self,
+        attributions: Union[Tensor, Tuple[Tensor, ...]],
+        is_attrib_tuple: bool,
+        return_convergence_delta: bool,
+        delta: Optional[float],
```

Comment:
This should actually be Optional[Tensor]. You will need to change line 274 above which initializes delta, to `delta = None`.